src/core/link-to-dfn.js

```diff
@@ -7,6 +7,9 @@ export const name = "core/link-to-dfn";
 export function run(conf, doc, cb) {
   doc.normalize();
   var titles = {};
+  var prevTitle;
+  var titleChanged = false;
+  var listOfDuplicateDfns = [];
```

Comment:
`prevTitle` is used to store the previous title so that when the title changes I can attach all the duplicate definitions related to that title all together
`titleChanged` keeps track of whether title changed or not
`listOfDuplicateDfns` used to store the list of duplicate definitions for a particular title